problems/arraysWithLoops.js

```diff
@@ -66,15 +128,40 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+function findSecondSmallest(nums) {
+  let smallNum = Infinity;
+  let secondSmall = Infinity;
+
+  for (let i = 0; i < nums.length; i++) {
+    if (nums[i] < smallNum) {
+      smallNum = nums[i];
+    }
+    for (let i = 0; i < nums.length; i++)
```

Comment:
Instead of nesting our loops, you can just use an else statement that checks for if num[i] < secondSmall because if it's hitting the else statement that already means it's greater than smallNum